src/recurrent/rnn.js

```diff
@@ -293,6 +293,21 @@ export default class RNN {
     this.ratioClipped = numClipped / numTot;
   }
 
+
+  /**
+   *
+   * @returns boolean
+   */
+  get isRunnable(){
+    if(this.model.equations.length === 0){
```

Comment:
nice

Reply:
This is one of the defining side effects of `train`, correct?